Bottom: df870f5bd477499bf8614e8308b36bc71dd18ab9
Top:    d64afafa5da6b2f33bf222ce50af979ef7f4f16e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 22:09:22 +0000

Refresh of attempt-to-fix-bad-rtree-usage

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 06cb7bc..48f34b7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1969,7 +1969,8 @@ print_labels (PLINE * a)
   do
     {
       DEBUGP ("(%mn, %mn)->(%mn, %mn) labeled %s  is_round=%i radius=%f cvc_next=%p cvc_prev=%p\n",
-	      EDGE_FORWARD_VERTEX (e)->point[0],  EDGE_FORWARD_VERTEX (e)->point[1], theState (e),
+              EDGE_BACKWARD_VERTEX (e)->point[0], EDGE_BACKWARD_VERTEX (e)->point[1],
+               EDGE_FORWARD_VERTEX (e)->point[0],  EDGE_FORWARD_VERTEX (e)->point[1], theState (e),
               e->is_round, e->radius, EDGE_BACKWARD_VERTEX (e)->cvc_next, EDGE_BACKWARD_VERTEX (e)->cvc_prev);
     }
   while ((e = NEXT_EDGE (e)) != &a->head);
